File: FibonacciProblems/FindingFibo.java
package FibonacciProblems;
public class FindingFibo {
    
    public static int f(int n) {
        if (n <= 1) {
            return n;
        }
        else
        {
            return f(n-1) + f(n-2);
        }
    }
    public static void main(String[] args) {
        int nthNumber = f(4);
        System.out.print(nthNumber);   
    }
}

File: FibonacciProblems/finding_fibo.py
def find_fibonacci(nthnumber:int):
    if nthnumber <= 1:
        return nthnumber
    else:
        return find_fibonacci(nthnumber - 1) + find_fibonacci(nthnumber - 2)

number_of_term = int(input("Enter number of term to print: "))
nthnumber = find_fibonacci(number_of_term)
print(nthnumber)